CMakeLists.txt

```diff
@@ -342,14 +342,8 @@ if(GDBSTUB)
   add_definitions(-DUSE_GDBSTUB)
 endif()
 
-if(VTUNE)
-  if(EXISTS "$ENV{VTUNE_AMPLIFIER_XE_2015_DIR}")
-    set(VTUNE_DIR "$ENV{VTUNE_AMPLIFIER_XE_2015_DIR}")
-  elseif(EXISTS "$ENV{VTUNE_AMPLIFIER_XE_2013_DIR}")
-    set(VTUNE_DIR "$ENV{VTUNE_AMPLIFIER_XE_2013_DIR}")
-  else()
-    message(ERROR "Could find neither VTUNE_AMPLIFIER_XE_2015_DIR nor VTUNE_AMPLIFIER_XE_2013_DIR.")
-  endif()
+if(ENABLE_VTUNE)
+  set(VTUNE_DIR "/opt/intel/vtune_amplifier")
```

Comment:
You should at least check if this directory exist, but I don't like to hard code a pth here.

Reply:
If we had a find module for VTune, `$ENV{VTUNE_AMPLIFIER_XE_2015_DIR}` and `$ENV{VTUNE_AMPLIFIER_XE_2013_DIR}` could be hint paths, while the lookup path is `/opt/intel/vtune_amplifier`, (and since it's done with `find_path`, it would allow overriding it on the command line if nothing could be found automatically).

Reply:
I really don't want to put that much effort into something that maybe one person uses every couple of years (it's been broken for three years).

Reply:
Oh I know, I can do that at some point. I was just saying what the alternative was